src/bidiTab/bidiTab.ts

```diff
@@ -52,21 +53,28 @@ declare global {
 
     // `window.setSelfTargetId` is called via `Runtime.evaluate` from the server side.
     setSelfTargetId: (targetId: string) => void;
+
+    // `window.verboseDebug` is set via `Runtime.evaluate` in case of verbose debugging mode.
+    verboseDebugChannel: string | undefined;
```

Comment:
https://google.github.io/styleguide/tsguide.html#optionals-vs-undefined-type:

> Use optional fields (on interfaces or classes) and parameters rather than a |undefined type.